Mathlib/CategoryTheory/Limits/Opposites.lean

```diff
@@ -401,112 +401,153 @@ theorem hasFiniteProducts_of_opposite [HasFiniteCoproducts Cᵒᵖ] : HasFiniteP
 
 section OppositeCoproducts
 
-variable {α : Type*} (Z : α → C) [HasCoproduct Z]
+variable {α : Type*} {Z : α → C} [HasCoproduct Z]
 
 instance : HasLimit (Discrete.functor Z).op := hasLimit_op_of_hasColimit (Discrete.functor Z)
 
-instance : HasLimit ((Discrete.opposite α).inverse ⋙ (Discrete.functor fun i ↦ Z i).op) :=
+instance : HasLimit ((Discrete.opposite α).inverse ⋙ (Discrete.functor fun i => Z i).op) :=
   hasLimitEquivalenceComp (Discrete.opposite α).symm
```

Comment:
`↦` is at least as good as `=>`: `↦` is actually more encouraged, so that you should revert such changes.

Reply:
I thought it was the other way around. I'll change it back.